### PR TITLE
CSI 4969 prepare before controller

### PR DESCRIPTION
**Pull request by ArbelNathan**

### CODE REVIEW

**Review by ELENAGER — APPROVED** [review]
LGTM

File: volume-group-operator/Makefile [hunk 1]
```diff
@@ -19,7 +19,6 @@ help: ## Display this help.
 manifests: controller-gen kustomize## Generate WebhookConfiguration, ClusterRole and CustomResourceDefinition objects.
 	$(CONTROLLER_GEN) $(CRD_OPTIONS) rbac:roleName=volume-group-operator webhook paths="./..." output:crd:artifacts:config=config/crd/bases
 	hack/update_labels_in_crd.sh
-	hack/update_labels_in_rbac.sh
```
Comment (matancarmeli7) [inline volume-group-operator/Makefile:22]: why did you remove this line?

File: volume-group-operator/config/crd/bases/csi.ibm.com_volumegroupclasses.yaml [hunk 2]
```diff
@@ -1,55 +1,73 @@
+---
 apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
```
Comment (matancarmeli7) [inline volume-group-operator/config/crd/bases/csi.ibm.com_volumegroupclasses.yaml:3]: how did you generate the yaml files? are you sure it passes correctly? because it looks like you skipped the `hack/update_labels_in_crd.sh` stage in the generation

File: volume-group-operator/Makefile (hunk 1 above)
Comment (ArbelNathan) [reply volume-group-operator/Makefile:22]: I didnt, can it be one of the scripts?

Comment (matancarmeli7) [reply volume-group-operator/Makefile:22]: which scripts? and GitHub shows that the line removed

Comment (ArbelNathan) [reply volume-group-operator/Makefile]: done

File: volume-group-operator/config/crd/bases/csi.ibm.com_volumegroupclasses.yaml (hunk 2 above)
Comment (ArbelNathan) [reply volume-group-operator/config/crd/bases/csi.ibm.com_volumegroupclasses.yaml]: done

**Review by matancarmeli7 — APPROVED** [review]
LGTM
